src/pages/index.js

```diff
@@ -254,7 +254,7 @@ const CustomArticle= (props) => {
 const MediumSection = () => {
   return (
     <div className="container section-container">
-       <h1 className="section-tittle">Artículos de Medium</h1>
+       <h1 className="section-tittle">Artículos de blogs</h1>
```

Comment:
Articulos de blogs o blog?

Reply:
Blog is correct, I wrote **blogs** to have a change to commit to try to test the netlify error, but in my last commit fixed, thank you for your observation